src/Platform/Microsoft.Testing.Platform/OutputDevice/TargetFrameworkParser.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+
+namespace Microsoft.Testing.Platform.OutputDevice;
+
+internal static class TargetFrameworkParser
+{
+    internal /* for testing */ static string? GetShortTargetFramework(string frameworkDescription)
```

Comment:
You could make it public as the class is `internal`. I usually go with `internal` on `internal` only if that's really not meant to be used.